agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -120,6 +120,16 @@ bool backhaul_manager::init()
         return false;
     }
 
+    if (m_sConfig.ucc_listener_port != 0) {
+        m_agent_ucc_listener = std::make_unique<agent_ucc_listener>(
+            *this, m_sConfig.ucc_listener_port, m_sConfig.vendor, m_sConfig.model,
+            m_sConfig.bridge_iface);
+        if (m_agent_ucc_listener && !m_agent_ucc_listener->start("ucc_listener")) {
+            LOG(ERROR) << "failed start agent_ucc_listener";
```

Comment:
We should print the error but not fail the whole agent init on this, IMO.

Reply:
I disagree. If an error occurs, there is no way to continue certification. So it is better to fail the init function so we won't be dragged into countless backhaul resets if something will go wrong (and it will).
Also, if the failure is due to the memory allocation failure, this alone should be failing the init.